tests/unit/composables/Project.spec.js

```diff
@@ -103,4 +139,41 @@ describe('Test composable: Project', () => {
       expect(projects).toStrictEqual({});
     });
   });
+
+  describe('Test function: initProject', () => {
```

Comment:
tests updateGitProject, not initGitProject

Reply:
done